spec/features/admin_sign_in_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+describe 'Admin sign in' do
+  let!(:member) { create :account }
+  let!(:account) { create :account }
+
+  it 'allows to sign in as admin' do
+    account.update(role: :admin)
```

Comment:
@gfedorenko Use factories: `let(:admin) { create :account, :admin)`. Read the docs for factory_bot.